src/clockwork/amqp.clj

```diff
@@ -30,4 +41,19 @@
     (log/info (format "[amqp/connect] [%s]" (config/amqp-uri)))
     (declare-exchange channel exchange-cfg)
     (declare-queue channel exchange-cfg queue-cfg (keys handlers))
-    (lc/blocking-subscribe channel (:name queue-cfg) (partial message-router handlers))))
+    (lc/blocking-subscribe channel (:name queue-cfg) (partial message-router handlers))
+    channel))
```

Comment:
(this might also make the code clearer, since it might avoid needing to use refs/ref-set -- but it might also just require doing that, but for the connection rather than the channel; dunno)